### PR TITLE
Add r-snftool.

### PR DESCRIPTION
**Pull request by cbrueffer**

### CODE REVIEW

File: recipes/r-snftool/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    - "\"%R%\" -e \"library('SNFtool')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=SNFtool
+  license: GPL
+  summary: Similarity Network Fusion takes multiple views of a network and fuses them together
+    to construct an overall status matrix. The input to our algorithm can be feature
+    vectors, pairwise distances, or pairwise similarities. The learned status matrix
+    can then be used for retrieval, clustering, and classification.
+  license_family: GPL
```
Comment (bgruening) [inline recipes/r-snftool/meta.yaml]: Can you add a license_file here?

**Comment by cbrueffer** [comment]
Same issue as with r-headmap.plus; license is generically "GPL" and no license file is shipped in the tarball.

File: recipes/r-snftool/meta.yaml (hunk 1 above)
Comment (bgruening) [reply recipes/r-snftool/meta.yaml]: Fair enough. Maybe we can add an issue to the feedstock and point upstream to it?

Comment (cbrueffer) [reply recipes/r-snftool/meta.yaml]: I'll try contacting them, maybe they can clarify.

**Comment by cbrueffer** [comment]
Could you commit pull/4594 as well?

**Pull request merged**